test/patch_require.rb

```diff
@@ -19,7 +19,7 @@ def require(name)
       case RUBY_PLATFORM
       when "x86_64-linux"
         suffix = '.so'
-      when "x86_64-darwin18", "x86_64-darwin19"
+      when "x86_64-darwin18", "x86_64-darwin19", "x86_64-darwin20"
```

Comment:
Where does this extra `darwin20` come from?

Reply:
Running tests locally on Big Sur :D